lib/workers/repository/update/branch/index.ts

```diff
@@ -115,17 +115,28 @@ export async function processBranch(
     // Check if branch already existed
     const existingPr = branchPr ? undefined : await prAlreadyExisted(config);
     if (existingPr && !dependencyDashboardCheck) {
-      logger.debug(
-        { prTitle: config.prTitle },
-        'Closed PR already exists. Skipping branch.'
-      );
-      await handlepr(config, existingPr);
-      return {
-        branchExists: false,
-        prNo: existingPr.number,
-        result: BranchResult.AlreadyExisted,
-      };
+      // Recreates pr if merged pr already exists
+      if (existingPr.state === 'merged') {
+        logger.debug(
+          { prTitle: config.prTitle },
+          'Merged PR already exists. Creating new PR with automerge disabled.'
```

Comment:
Only mention automerge disabled if automerge was enabled in the first place, otherwise it could confuse

Reply:
Use two debug log messages if necessary